Lib/argparse.py

```diff
@@ -263,7 +263,7 @@ def add_argument(self, action):
                 invocations.append(get_invocation(subaction))
 
             # update the maximum item length
-            invocation_length = max([len(s) for s in invocations])
+            invocation_length = max(len(s) for s in invocations)
```

Comment:
Let's skip this module.  I think Bethard would like this to be compatible across as many versions as possible or at least to keep the code in-sync as much as possible.